ApplicationLibCode/ReservoirDataModel/RigMainGrid.cpp

```diff
@@ -433,6 +435,35 @@ bool RigMainGrid::hasFaultWithName( const QString& name ) const
     return false;
 }
 
+//--------------------------------------------------------------------------------------------------
+///
+//--------------------------------------------------------------------------------------------------
+void RigMainGrid::computeBoundingBox()
+{
+    m_boundingBox = cvf::BoundingBox();
```

Comment:
Maybe `m_boundingBox.reset(); `communicates better, but not sure.

Reply:
agree